examples/react-es6/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "react": "0.13.0-beta.1",
+    "react-tools": "0.13.0-beta.1"
```

Comment:
We're at RC2 now. Perhaps we should wait until we hit final to merge this (and point at that)